openedx/features/content_type_gating/templates/content_type_gating/access_denied_message.html

```diff
@@ -0,0 +1,19 @@
+<div class="content-paywall">
+    <div>
+        <h3>
+            <span class="fa fa-lock" aria-hidden="true"></span>
+            Verified Track Access
+        </h3>
+        <span style=" padding: 10px 0;">
+            Graded assessments are available to Verified Track learners.
+        </span>
+        {% if not mobile_app %}
+        <span class="certDIV_1" style="">
+            <a href="{{ecommerce_checkout_link}}" class="certA_2">
+                Upgrade to unlock  (${{min_price}})
+            </a>
+        </span>
+        {% endif %}
+    </div>
+    <img src="https://courses.edx.org/static/images/edx-verified-mini-cert.png">
```

Comment:
I feel like this should be a relative path, so that non-edX installations aren't using our edX hosted static assets.

Reply:
Fixed